src/main/java/ua/ithillel/lesson5/MyArrayList.java

```diff
@@ -0,0 +1,69 @@
+package ua.ithillel.lesson5;
+
+public class MyArrayList implements HillelList{
+
+    private String[] baseArray;
```

Comment:
```suggestion
    private String[] baseArray = new String[0];
```
тоді не потрібно постійно перевіряти на null